[DOCS] Update Cloud Snowflake install instructions

Kilo59 opened the pull request:
This PR updates the Snowflake Cloud Quickstart instructions to ensure that the required `snowflake-sqlalchemy` dependency is installed. Otherwise users will get an error if the try to actually connect to their Snowflake Datasource.

Alternatively we could update the instructions as follows.
```shell
pip install 'great_expectations[cloud,snowflake]'
```

Kilo59 committed:
suggest users pip install `great_expectations_cloud[snowflake]`

Kilo59 commented on docs/docusaurus/docs/cloud/set_up_gx_cloud.md:
````diff
@@ -28,15 +28,15 @@ To get the most out of GX Cloud, you'll need to request a GX Cloud Beta account,
 
     ```bash title="Terminal input"
-    pip install great_expectations[cloud]
+    pip install 'great_expectations_cloud[snowflake]'
````
```suggestion
    pip install 'great_expectations[cloud,snowflake]'
```
Alternative instructions

Kilo59 commented on docs/docusaurus/docs/cloud/set_up_gx_cloud.md:
````diff
@@ -28,15 +28,15 @@ To get the most out of GX Cloud, you'll need to request a GX Cloud Beta account,
 
     ```bash title="Terminal input"
-    pip install 'great_expectations[cloud]' --upgrade
+    pip install 'great_expectations_cloud[snowflake]' --upgrade
````
```suggestion
    pip install 'great_expectations[cloud,snowflake]' --upgrade
```
Alternative instructions

ghost commented:
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=8776&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/great-expectations/great_expectations/8776/514bce47/a8411f8818b91d8ddc793ffa7810ce90d10b1c85.svg)](https://app.codesee.io/r/reviews?pr=8776&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
<img src="https://app.codesee.io/Legend_v3.png" alt="CodeSee Map legend" width="220" height="156">
</details>

kwcanuck committed:
Apply suggestions from code review

LGTM. Odd - the process worked for me as written. I must have the dependencies installed already.

Co-authored-by: Gabriel <gabriel59kg@gmail.com>
